src/AddTodoForm.jsx

```diff
@@ -11,9 +11,8 @@ function AddTodoForm({ onAddTodo }) {
   };
 
   const handleAddTodo = (event) => {
-    event.preventDefault();
```

Comment:
because you removed event.preventDefault you cannot add new todo

Reply:
Fixed! Here is the commit: https://github.com/nihalerdal/react-todo/commit/f94100320e9b20c753b4226f12a04f19fe80b362